src/Avalonia.Base/Layout/MinMax.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace Avalonia.Layout;
+
+internal struct MinMax
```

Comment:
I know this was ported from WPF and is internal, but the naming doesn't seem right to me.

Instead of `MinMax` I would expect something like `Range`.

However, the best name is something more like `SizeConstraints`.